src/game.py

```diff
@@ -42,10 +45,19 @@ def place_mark_on_board(self, user_input, board, play_count):
     def process_user_input(self):
         position_choice = int(input())
         self.place_mark_on_board(position_choice, self.board, self.play_count)
+        self.play_count += 1
         print(self.initialize_board())
 
+    def progress_game(self):
+        current_play_count = self.count_plays()
+        while current_play_count != len(self.board):
+            print(self.get_prompt(current_play_count))
+            self.process_user_input()
+            current_play_count += 1
+        else:
+            print(self.get_prompt(current_play_count))
```

Comment:
Do you have test coverage for this? It looks like a new method, but I don't see new tests in this branch.

Reply:
This is one I wasn't sure how to add test coverage for because it uses my method that prompts an input on the command line.  So maybe that's indicative that I should reconsider it...?